src/pages/home/report/ReportActionCompose.js

```diff
@@ -165,1165 +178,1167 @@ const getMaxArrowIndex = (numRows, isAutoSuggestionPickerLarge) => {
 
     /**
-     * Returns the list of IOU Options
-     *
-     * @param {Array<Number>} reportParticipants
-     * @returns {Array<object>}
+     * Focus the composer text input
+     * @param {Boolean} [shouldDelay=false] Impose delay before focusing the composer
+     * @memberof ReportActionCompose
      */
-    getMoneyRequestOptions(reportParticipants) {
-        const options = {
-            [CONST.IOU.MONEY_REQUEST_TYPE.SPLIT]: {
-                icon: Expensicons.Receipt,
-                text: this.props.translate('iou.splitBill'),
-            },
-            [CONST.IOU.MONEY_REQUEST_TYPE.REQUEST]: {
-                icon: Expensicons.MoneyCircle,
-                text: this.props.translate('iou.requestMoney'),
-            },
-            [CONST.IOU.MONEY_REQUEST_TYPE.SEND]: {
-                icon: Expensicons.Send,
-                text: this.props.translate('iou.sendMoney'),
-            },
-        };
-        return _.map(ReportUtils.getMoneyRequestOptions(this.props.report, reportParticipants, this.props.betas), (option) => ({
-            ...options[option],
-            onSelected: () => IOU.startMoneyRequest(option, this.props.report.reportID),
-        }));
-    }
+    const focus = useCallback((shouldDelay = false) => {
+        // There could be other animations running while we trigger manual focus.
+        // This prevents focus from making those animations janky.
+        InteractionManager.runAfterInteractions(() => {
+            if (!textInput.current) {
+                return;
+            }
+
+            if (!shouldDelay) {
+                textInput.current.focus();
+            } else {
+                // Keyboard is not opened after Emoji Picker is closed
+                // SetTimeout is used as a workaround
+                // https://github.com/react-native-modal/react-native-modal/issues/114
+                // We carefully choose a delay. 100ms is found enough for keyboard to open.
+                setTimeout(() => textInput.current.focus(), 100);
+            }
+        });
+    }, []);
 
     /**
-     * Updates the composer when the comment length is exceeded
-     * Shows red borders and prevents the comment from being sent
+     * Update the value of the comment in Onyx
      *
-     * @param {Boolean} hasExceededMaxCommentLength
+     * @param {String} comment
+     * @param {Boolean} shouldDebounceSaveComment
      */
-    setExceededMaxCommentLength(hasExceededMaxCommentLength) {
-        this.setState({hasExceededMaxCommentLength});
-    }
+    const updateComment = useCallback(
+        (commentValue, shouldDebounceSaveComment) => {
+            const {text: newComment = '', emojis = []} = EmojiUtils.replaceEmojis(commentValue, preferredSkinTone, preferredLocale);
+
+            if (!_.isEmpty(emojis)) {
+                User.updateFrequentlyUsedEmojis(EmojiUtils.getFrequentlyUsedEmojis(emojis));
+                insertedEmojis.current = [...insertedEmojis, ...emojis];
```

Comment:
I think this needs to use `.current` as its a ref (I am also getting warnings in my browser when typing otherwise)